src/components/AppNavigationItem/AppNavigationItem.vue

```diff
@@ -506,8 +506,8 @@ export default {
 	box-sizing: border-box;
 	width: 100%;
 	min-height: $clickable-area;
-	padding-right: 8px;
-	border-radius: var(--border-radius-large);
+	transition: background-color 200ms ease-in-out;
```

Comment:
should use either of:
```
  --animation-quick: 100ms;
  --animation-slow: 300ms;
```

So in the future it works with an option for a11y animations out of the box?